container-build/cerc-fixturenet-eth-geth/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM skylenet/ethereum-genesis-generator AS ethgen
+FROM 210353ce7c898686bc5092f16c61220a76d357f51eff9c451e9ad1b9ad03d4d3 AS ethgen # skylenet/ethereum-genesis-generator
```

Comment:
Should it be like: 
`FROM skylenet/ethereum-genesis-generator@sha256:210353ce7c898686bc5092f16c61220a76d357f51eff9c451e9ad1b9ad03d4d3` ?

Reply:
Let me find out.  I hope so, because it's easier to read.   My first attempt was skylenet/ethereum-genesis-generator@210353ce7c898686bc5092f16c61220a76d357f51eff9c451e9ad1b9ad03d4d3, but that just got interpreted as a tag...

Reply:
Hurray!  I also updated the reference to `sigp/lcli` to use an explicit version.